<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogQuickAction</class>
 <widget class="QDialog" name="DialogQuickAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick Action</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxScenery">
     <property name="title">
      <string>Scenery</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelScenery">
        <property name="text">
         <string>Scenery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxScenery</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxScenery"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxOwnship">
     <property name="title">
      <string>Ownship</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTypeOwnship">
        <property name="text">
         <string>Aircraft Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxTypeOwnship</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxTypeOwnship"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLiveryOwnship">
        <property name="text">
         <string>Livery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxLiveryOwnship</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLiveryOwnship"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxOpponent_1">
     <property name="title">
      <string>Opponent 1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTypeOpponent_1">
        <property name="text">
         <string>Aircraft Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxTypeOpponent_1</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxTypeOpponent_1"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLiveryOpponent_1">
        <property name="text">
         <string>Livery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxLiveryOpponent_1</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLiveryOpponent_1"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxWingman">
     <property name="title">
      <string>Wingman</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTypeWingman">
        <property name="text">
         <string>Aircraft Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxTypeWingman</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxTypeWingman"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLiveryWingman">
        <property name="text">
         <string>Livery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxLiveryWingman</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLiveryWingman"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBoxOpponent_2">
     <property name="title">
      <string>Opponent 2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTypeOpponent_2">
        <property name="text">
         <string>Aircraft Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxTypeOpponent_2</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxTypeOpponent_2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLiveryOpponent_2">
        <property name="text">
         <string>Livery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxLiveryOpponent_2</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLiveryOpponent_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBoxAlly_1">
     <property name="title">
      <string>Ally 1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTypeAlly_1">
        <property name="text">
         <string>Aircraft Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxTypeAlly_1</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxTypeAlly_1"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLiveryAlly_1">
        <property name="text">
         <string>Livery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxLiveryAlly_1</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLiveryAlly_1"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBoxOpponent_3">
     <property name="title">
      <string>Opponent 3</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTypeOpponent_3">
        <property name="text">
         <string>Aircraft Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxTypeOpponent_3</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxTypeOpponent_3"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLiveryOpponent_3">
        <property name="text">
         <string>Livery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxLiveryOpponent_3</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLiveryOpponent_3"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBoxAlly_2">
     <property name="title">
      <string>Ally 2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTypeAlly_2">
        <property name="text">
         <string>Aircraft Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxTypeAlly_2</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxTypeAlly_2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLiveryAlly_2">
        <property name="text">
         <string>Livery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxLiveryAlly_2</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLiveryAlly_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBoxOpponent_4">
     <property name="title">
      <string>Opponent 4</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTypeOpponent_4">
        <property name="text">
         <string>Aircraft Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxTypeOpponent_4</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxTypeOpponent_4"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLiveryOpponent_4">
        <property name="text">
         <string>Livery:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboBoxLiveryOpponent_4</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxLiveryOpponent_4"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>408</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxScenery</tabstop>
  <tabstop>comboBoxTypeOwnship</tabstop>
  <tabstop>comboBoxLiveryOwnship</tabstop>
  <tabstop>comboBoxTypeWingman</tabstop>
  <tabstop>comboBoxLiveryWingman</tabstop>
  <tabstop>comboBoxTypeAlly_1</tabstop>
  <tabstop>comboBoxLiveryAlly_1</tabstop>
  <tabstop>comboBoxTypeAlly_2</tabstop>
  <tabstop>comboBoxLiveryAlly_2</tabstop>
  <tabstop>comboBoxTypeOpponent_1</tabstop>
  <tabstop>comboBoxLiveryOpponent_1</tabstop>
  <tabstop>comboBoxTypeOpponent_2</tabstop>
  <tabstop>comboBoxLiveryOpponent_2</tabstop>
  <tabstop>comboBoxTypeOpponent_3</tabstop>
  <tabstop>comboBoxLiveryOpponent_3</tabstop>
  <tabstop>comboBoxTypeOpponent_4</tabstop>
  <tabstop>comboBoxLiveryOpponent_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogQuickAction</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogQuickAction</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
